src/primitives/transaction.h

```diff
@@ -971,6 +971,19 @@ class CTransaction
     // Return sum of (positive valueBalanceSapling or zero) and JoinSplit vpub_new
```

Comment:
The documentation here and for `GetValueOut` does not take into account Orchard, even though iirc the implementations *were* changed for Orchard.